e2e2/test/images/nvidia/Dockerfile

```diff
@@ -97,4 +102,11 @@ RUN rm -rf /var/lib/apt/lists/*
 ENV LD_PRELOAD /usr/lib/x86_64-linux-gnu/libnccl.so:$LD_PRELOAD
 
 COPY e2e2/test/images/nvidia/gpu_unit_tests ./gpu_unit_tests
-RUN chmod +x ./gpu_unit_tests/unit_test
\ No newline at end of file
+RUN chmod +x ./gpu_unit_tests/unit_test
+
+# Print the values of EFA_INSTALLER_VERSION and NCCL_VERSION
+CMD echo "EFA Installer Version: $EFA_INSTALLER_VERSION" && \
+    echo "NCCL Version: $NCCL_VERSION" && \
+    echo "AWS OFI NCCL Version: $AWS_OFI_NCCL_VERSION" && \
+    printf "NVIDIA Driver Version: " && \
+    nvidia-smi --query-gpu=driver_version --format=csv,noheader | head -n 1
```

Comment:
I think this would be better suited for an `ENTRYPOINT` script that logged this info and then ran whatever `CMD` was used

Reply:
Updated in the latest revision